### ISSUE TITLE
Skip execution of Maven plugin

### ISSUE THREAD

**Issue by flo-ryan** [smallrye/smallrye-open-api#558]
Is there a skip parameter or something similar for the Maven plugin, as there is for many other Maven plugins?

This does not seem to work:
```
<groupId>io.smallrye</groupId>
<artifactId>smallrye-open-api-maven-plugin</artifactId>
<version>2.0.14</version>
<executions>
	<execution>
		<phase>compile</phase>
		<goals>
			<goal>generate-schema</goal>
		</goals>
		<configuration>
			<skip>true</skip>
		</configuration>
	</execution>
</executions>
```

**Comment by phillip-kruger** [comment]
No there is not, but it should be easy enough to add. Care to provide a PR ? :)

**Comment by flo-ryan** [comment]
I haven't worked on a Maven plugin yet, but I can give it a try. I'll look into it over the weekend.

**Comment by MikeEdgar** [comment]
Fixed via #559, #560

**Comment by flo-ryan** [comment]
I see I was too slow ;) Thanks @phillip-kruger!
